YYMM-office-hours-recap-template.md

```diff
@@ -0,0 +1,19 @@
+Title: "<Title Here>" - DEI WG Community Office Hour -
+
+Body:
+<Heading>
+
+Summary
+<Summary>
+
+Topics Discussed
+1. <First Topic>
+2. <Second Topic>
+3. <Third Topic>
+
+Learnings Observed
+Mentees can ...
+
+Maintainers can ...
+
+Learn more today: Visit OpenSSF DEI Working Group - hyperlink
```

Comment:
I think the link to the youtube recording will be more visible if we put it at the top right under the title. We can point to the Slack channel here at the bottom?

Reply:
I like this change. Itd be nice to be able to transition from the document to the YouTube video from GitHub.